src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -103,7 +108,7 @@ export class HaIntegrationCard extends LitElement {
             >
               <ha-list-item hasMeta graphic="icon">
                 <ha-svg-icon
-                  .path=${services ? mdiApplication : mdiDevices}
+                  .path=${services ? mdiHandExtendedOutline : mdiDevices}
```

Comment:
Not sure about this icon but it's just personal thought.

Reply:
yeah 🤷‍♂️